app/models/countries.rb

```diff
@@ -0,0 +1,247 @@
+# This list is taken from https://www.gov.uk/api/foreign-travel-advice.json
+# $ curl https://www.gov.uk/api/foreign-travel-advice.json | jq '.details.countries[].name'
+
+module Countries
+  def self.all
```

Comment:
I prefer bare-words to direct constant access, scope is more predictable, and it is simpler to extend.

I made the constant private to make it unambiguous that the methods should be used.

And I didn't use freeze because freezing an Array doesn't freeze its elements, so using it here would be misleading.